README.md

```diff
@@ -221,11 +221,16 @@ threading restrictions.
 
 Thus, the following are not supported:
 
-- Android
+- Android (consider [Firebase](https://firebase.google.com), which includes many of these APIs)
 - Alpine Linux (due to netty-tcnative requiring glibc, which is not present on Alpine)
 - Raspberry Pi (since it runs on the ARM architecture)
 - Google App Engine Standard Java 7
 
+*Android users*: Although unsupported, it is possible to use these libraries in many cases.
```

Comment:
I think I'd prefer this to be a subbullet of the Android bullet above

Reply:
sure, I moved it.